src/wattsi.pas

```diff
@@ -681,11 +686,61 @@    TCrossReferences = record
                            Scratch.Append($002E);
                         Scratch.Append(IntToStr(CurrentSectionNumber[CurrentHeadingRank]));
                      end;
+                     if (Variant = vDEV) then
+                     begin
+                        ID := Element.GetAttribute('id').AsString;
+                        SectionNumber := Scratch.AsString;
+                        HeadingText := DelSpace1(StringReplace(StringReplace(Element.TextContent.AsString, #$0A, ' ', [rfReplaceAll]), #$22, '\' + #$22, [rfReplaceAll]));
```

Comment:
(StringReplace(StringReplace(Element.TextContent.AsString, #$0A, ' ', [rfReplaceAll]), #$22, '\' + #$22, [rfReplaceAll]));

> Can you comment on what this line is doing

Comment added. The only *necessary* thing it does is to escape the quote marks. The extra stuff it’s doing is just to change newlines into spaces, and to squash all sequences of multiple space into a single space. https://www.freepascal.org/docs-html/rtl/strutils/delspace1.html

> Maybe it should even be factored out into a function as I see it's repeated a bit below.

Yes, made it into `MungeForJsonOutput`